include/circt/Conversion/ConvertToArcs.h

```diff
@@ -13,7 +13,13 @@
 #include <memory>
 
 namespace circt {
-std::unique_ptr<OperationPass<ModuleOp>> createConvertToArcsPass();
+
+#define GEN_PASS_DECL_CONVERTTOARCS
+#include "circt/Conversion/Passes.h.inc"
+
+std::unique_ptr<OperationPass<ModuleOp>>
+createConvertToArcsPass(const ConvertToArcsOptions &options = {});
```

Comment:
nit: This should be automatically generated if we don't define `constructor` in ODS (Passes.td Line 615)

Reply:
I did not know that was a thing. That's fantastic! Feels like we can get rid of almost all of those constructors that way 😍

Reply:
Hmmm, the auto-generated constructor is just called `createConvertToArcs` :crying_cat_face: So we'd have to go through all pass names defined in tablegen files and add a "Pass" suffix.